eventstore/tasks.py

```diff
@@ -626,7 +626,7 @@ def get_turn_profile_link(contact_number):
         }
 
         if contact_number:
-            turn_url = settings.TURN_URL + "/v1/contacts/{}/messages".format(
+            turn_url = settings.TURN_URL + "v1/contacts/{}/messages".format(
```

Comment:
Let rather use the `urljoin` package to build this URL, then we'll know it will always work, doesn't matter what TURN_URL is configured as.

Reply:
Example: https://github.com/praekeltfoundation/ndoh-hub/blob/develop/eventstore/tasks.py#L71